main.go

```diff
@@ -93,8 +95,12 @@ func main() {
 		return
 	}
 
-	if err := config.ParseConfig(*configPath); err != nil {
-		log.Fatalf("Error reading config: %v", err)
+	if !filepath.IsAbs(*configPath) {
+		if err := config.ParseConfig(*configPath); err != nil{
+			log.Fatalf("Error reading config: %v", err)
+		}
+	} else {
+		config.SetMysqlUnixUser(*dbUser)
```

Comment:
Why not to read the config in this case?

Reply:
I don't really understand what you mean?

Reply:
I mean: now if `filepath.IsAbs(*configPath)` you don’t `config.ParseConfig(*configPath)`.

Reply:
Because ``ParseConfig()`` parses only ``*.ini`` files and not ``*.socket`` files.

Reply:
No one ever said anything about not parsing ini files, nor "parsing socket files".

Reply:
... so why you’re doing this?